src/views/MessageRepliesView.tsx

```diff
@@ -19,7 +19,7 @@ export default function MessageRepliesView({
       {replies.length > 0 && (
         <div className="mb-2">
           {replies.map((message) => (
-            <div className="flex text-xs space-x-1">
+            <div className="flex text-xs space-x-1" key={message.id}>
```

Comment:
unrelated, but React was complaining about this

Reply:
We could probably use `message.xmtpID` instead.
```suggestion
            <div className="flex text-xs space-x-1" key={message.xmtpID}>
```